pkg/pillar/nireconciler/linux_config.go

```diff
@@ -134,6 +134,20 @@ import (
 //  |   |   +----------------------------------------------------+   |   |
 //  |   |                                                            |   |
 //  |   |   +----------------------------------------------------+   |   |
+//  |   |   |            Mirroring (for Switch NI)               |   |   |
+//  |   |   |                                                    |   |   |
+//  |   |   |            +------------------------+              |   |   |
+//  |   |   |            |        DummyIf         |              |   |   |
```

Comment:
We already have a dummyIf for the ACL drops. I assume this is a different dummy. If so, shouldn't we give them specific names? AclDropIf and MirrorDropIf?

Reply:
In this graph I describe the graph-based representation of the configuration, in particular all the sugraphs and the config item types used. So here I only put type names (IPSet, IPRule, DummyIf) which are being used and potentially describe the use-case in brackets below (added it now for the "drop" dummy).

Reply:
I do it like this because most of those config items are used with many instances (e.g. multiple routes, IP sets, iptables rules), etc., so naming here would only make sense for singleton items.

Reply:
Sorry missed your comment over the weekend. In the implementation is this the same dummy interface as we use for ACL drops? Or a different interface?

Reply:
Different interface - actually interfaces - one dummy (named `<bridge>-m`) for every NI with active packet sniffing (switch NIs and Local NIs with flow logging enabled). I do not want to mix packets mirrored from different NIs, even though in theory it should not create problems in terms of patching captured packet to a source/destination app. Also, I did not want to change the code too much and currently we have one pcap Go routine running separately for every NI with active packet sniffing (so I only change the target for pcap from the bridge to the corresponding dummy interface).